### PR TITLE
maponly_6

### PR DESCRIPTION
**Pull request by erinlincoln**
- Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/maponly_6.ncl
- Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/maponly_6_lg.png
- Issue #374 
maponly_6_fig_1

### CODE REVIEW

**Review by Jiaqi-beep — COMMENTED** [review]
Looks really good! It seems to me that the state and lake boundaries in the original plot is thinner. The other thing to match original plot (that might not be necessary) could be setting the lake boundaries to translucent for the great lakes.

File: Plots/MapProjections/NCL_maponly_6.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                          linewidth=0.7)
+
+# Create colorbar based on mapped colormap norm
+fig.colorbar(mappable=mappable,
+             cax=axins1,
+             boundaries=colorbounds,
+             ticks=colorbounds[1:-1],
+             spacing='uniform',
+             orientation='horizontal',
+             anchor=(0.1, 0.5))
```
Comment (anissa111) [inline Plots/MapProjections/NCL_maponly_6.py:131]:
```suggestion
fig.colorbar(mappable=mappable,
             cax=axins1,
             boundaries=colorbounds,
             ticks=colorbounds[1:-1],
             spacing='uniform',
             orientation='horizontal',
             drawedges=True,
             anchor=(0.1, 0.5))
```

**Review by anissa111 — COMMENTED** [review]
This looks good, I just left one comment below.

**Review by anissa111 — APPROVED** [review]
Looks good!

**Pull request merged**